doc/template.html

```diff
@@ -23,7 +23,24 @@
 
     <div id="column1" data-id="__ID__" class="interior">
       <header>
-        <h1>Node.js __VERSION__ Documentation</h1>
+        <div class="header-container">
+          <h1>Node.js __VERSION__ Documentation</h1>
+          <button class="theme-toggle-btn" id="theme-toggle-btn">
+            <span class="icon dark-icon">
+              <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24">
```

Comment:
Let's remove the extra `<span>` elements that are not useful I believe.
```suggestion
            <svg xmlns="http://www.w3.org/2000/svg" class="icon dark-icon" height="24" width="24">
```